README.rst

```diff
@@ -42,50 +42,98 @@ You use a template to create a cheat sheet. Here is the procedure to follow:
-This command uses latexmk to compile your TeX file into a PDF, and then convert the first page of the PDF into a PNG file.
+   Here, replace ``<your_project>``, ``<your_folder_name>``, and ``<your_tex_file_name>`` with your actual project name, folder name, and TeX file name, respectively.
+   This command uses latexmk to compile your TeX file into a PDF, and then convert the first page of the PDF into a PNG file.
 
 #.  In ``makefile``, add this command to ``make all``.
 
+**For Windows users**
+
+#. In ``make.bat``, add a new makefile command:
+
+   .. code:: bash
+
+         :<your_project>_cheat_sheet
+            pdflatex -output-directory=%BUILDDIR% cheat_sheats/<your_folder_name>/<your_tex_file_name>.tex --interaction=nonstopmode
+            goto end
+
+   Here, replace ``<your_project>``, ``<your_folder_name>``, and ``<your_tex_file_name>`` with your actual project name, folder name, and TeX file name, respectively.
+   This command uses latexmk to compile your TeX file into a PDF.
+
+#.  In ``make.bat``, add this command to ``all``.
+
+Add cheat sheet in landing page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In ``index.rst`` in ``doc/source``, Add new grid item card for the new cheatsheet by adding command:
+
+
+.. code:: bash
+
+   .. grid-item::
+
+           .. card:: <Your_project_name>
+               :img-top: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.png
+               :link: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.pdf
+
+
+Finally add the screenshot on readme with proper links and references.
+
+
 Cheat sheets
 ~~~~~~~~~~~~
 
-`download PyMAPDL cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.pdf>`_
-
-.. image:: https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.png
++------------------------------------+-------------------------------------+
+| `Download PyMAPDL cheatsheet pdf`_ | `Download PyFluent cheatsheet PDF`_ |
++====================================+=====================================+
+| |pymapdl cheatsheet snapshot|      |  |pyfluent cheatsheet snapshot|     |
++------------------------------------+-------------------------------------+
+
++---------------------------------------+--------------------------------------+
+| `Download PyAEDT API cheatsheet pdf`_ | `Download PyEDB API cheatsheet pdf`_ |
++=======================================+======================================+
+| |pyaedt API cheatsheet snapshot|      |   |pyedb API cheatsheet snapshot|    |
++---------------------------------------+--------------------------------------+
+
++--------------------------------------------+-----+
+| `Download PyPrimeMesh API cheatsheet pdf`_ |     |
++============================================+=====+
+| |PyPrimeMesh cheatsheet snapshot|          |     |
++--------------------------------------------+-----+
+
```

Comment:
The rendering of this table is not great. Please work on it so that they have a similar distribution